components/ArtworkArticle/ArtworkDetails.js

```diff
@@ -0,0 +1,13 @@
+export default function ArtworkDetails({ data }) {
+  const { dimensions, technique, price } = data;
+  return (
+    <section className="artwork-details__details-container">
+      <p>Técnica:</p>
+      <p>{dimensions}</p>
+      <p>Dimensiones:</p>
+      <p>{technique}</p>
```

Comment:
Here the values are inverted, technique and dimensions

Reply:
Wow! Nice catch lol thank you!